dev/com.ibm.ws.rest.handler.config_fat/publish/servers/com.ibm.ws.rest.handler.config.openapi.fat/jvm.options

```diff
@@ -0,0 +1 @@
+-Dcom.ibm.ws.beta.edition=true
```

Comment:
I don't think this is necessary unless we have beta guards in code. The FAT tests run with all features available anyway.